pkg/scheduler/metrics/metrics.go

```diff
@@ -189,8 +189,16 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"plugin", "profile"})
 
+	ScheduleAttempts = metrics.NewCounterVec(
```

Comment:
counters should always end in `_total` https://prometheus.io/docs/practices/naming/#metric-names

`plugin_evaluation_total`? This is a low level metric, so I think it's fair to have the word plugin in it.

Reply:
Renamed as suggested, thanks.